pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -161,9 +161,6 @@ cpdef convert_to_timedelta64(object ts, object unit):
         - None/NaT
 
     Return an ns based int64
-
-    # kludgy here until we have a timedelta scalar
-    # handle the numpy < 1.7 case
```

Comment:
Do you know what section of code this was referring to?

Reply:
@TomAugspurger 
I don't see something particularly kludge-y either. Also, there is a timedelta scalar nowadays. It just seems like a severely outdated comment.